pkg/virt-launcher/virtwrap/api/schema.go

```diff
@@ -323,6 +323,8 @@ type HostDevice struct {
 	Type      string           `xml:"type,attr"`
 	BootOrder *BootOrder       `xml:"boot,omitempty"`
 	Managed   string           `xml:"managed,attr"`
+	Mode      string           `xml:"mode,attr,omitempty"`
```

Comment:
I see that we don't define an `Alias` for GPU and SRIOV. We should actually pass through the `name` from the VMI definition, to allow tracking the devices throughout the stack.

Reply:
@rmohr considering it's already an issue for SR-IOV devices, should we perhaps leave it out of this PR? I am happy to take it on myself to address for both. Let me know.

Reply:
@rmohr and @booxter Let me know if you want to fix it as a part of different PR

Reply:
Extra PR is fine. Let's ensure we tackle it soon though.

Reply:
@rmohr thanks. I made a note in my todo list so that it doesn't fall through cracks.